package.json

```diff
@@ -39,6 +39,7 @@
     "inflection": "^1.6.0",
     "lodash": "^3.9.3",
     "moment": "^2.11.0",
+    "microtime": "^2.0.0",
```

Comment:
microtime requires gyp and uses gettimeofday. It won't work on windows, question is whether or not it will install.